src/featbin/wav-reverberate-test.cc

```diff
@@ -0,0 +1,143 @@
+// featbin/extract-segments.cc
```

Comment:
Either feat/ or matrix/ are suitable places to put convolution code in
principle,  but the bar is higher if you want to include them in matrix/.
That is, you'd need to think carefully about the interface and make sure
that the code handles all edge cases in a good way, does a reasonable thing
for all sizes of input, and is tested to a high standard.  It's your choice
what you want to do.
Dan


On Mon, Aug 24, 2015 at 10:32 PM, Kirill Katsnelson <
notifications@github.com> wrote:

> In src/featbin/wav-reverberate-test.cc
> <https://github.com/kaldi-asr/kaldi/pull/76#discussion_r37834039>:
>
> > @@ -0,0 +1,143 @@
> > +// featbin/extract-segments.cc
>
> @vijayaditya <https://github.com/vijayaditya>: perhaps it makes sense to
> put these DSP functions in one of the libraries, either matrix/ (this is
> where we have SplitRadixRealFft) or feat/. @danpovey
> <https://github.com/danpovey>, what would you recommend? We do not run
> tests in *bin/ directories.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/76/files#r37834039>.
>